qfieldsync/utils/file_utils.py

```diff
@@ -61,3 +62,27 @@ def path_to_dict(path: PathLike, dirs_only: bool = False) -> DirectoryTreeDict:
     node["content"].sort(key=lambda node: node["path"].name)
 
     return node
+
+
+def is_valid_filename(filename: str) -> bool:
+    """
+    Check if the filename is valid.
+    """
+    pattern = re.compile(
+        r'\A(?!(?:COM[0-9]|CON|LPT[0-9]|NUL|PRN|AUX|com[0-9]|con|lpt[0-9]|nul|prn|aux)|\s|[\.]{2,})[^\\\/:*"?<>|]{1,254}(?<![\s\.])\z'
+    )
+    return bool(pattern.match(filename))
+
+
+def is_valid_path(path: str) -> bool:
```

Comment:
```suggestion
def is_valid_filepath(path: str) -> bool:
```

Can we use the same name as in the quoted library?

Reply:
Yes for sure, is way more better!

Reply:
Great!